### PR TITLE
Announce the schedule of 2019 Spark+AI summit at SF

### PR DESCRIPTION
**Pull request by gatorsmile**
screen shot 2018-12-19 at 4 59 12 pm
screen shot 2018-12-19 at 4 59 02 pm

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @rxin @yhuai @cloud-fan @srowen

File: site/mailing-lists.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   
     <meta http-equiv="refresh" content="0; url=/community.html">
-    <link rel="canonical" href="https://spark.apache.org/community.html" />
+    <link rel="canonical" href="http://localhost:4000/community.html" />
```
Comment (gatorsmile) [inline site/mailing-lists.html]: Need to fix this.

File: site/sitemap.xml [hunk 2]
```diff
@@ -139,657 +139,661 @@
 </url>
 <!-- Auto-generate sitemap for rest of site content -->
 <url>
-  <loc>https://spark.apache.org/releases/spark-release-2-4-0.html</loc>
+  <loc>http://localhost:4000/news/spark-ai-summit-apr-2019-agenda-posted.html</loc>
```
Comment (ueshin) [inline site/sitemap.xml]: Still remaining `localhost:4000` in this file.

**Review by srowen — APPROVED** [review]
Seems fine to me if there are no references to localhost. That tends to happen if you run `jekyll serve` locally.

**Comment by gatorsmile** [comment]
Thanks! Merged to master.

File: site/sitemap.xml (hunk 2 above)
Comment (ueshin) [reply site/sitemap.xml]: @gatorsmile oops, I mean, there are a lot of `localhost:4000` in this file.

Comment (gatorsmile) [reply site/sitemap.xml]: Will update them later.

Comment (srowen) [reply site/sitemap.xml]: I just pushed a fix directly, it's done

**Pull request closed without merging**